SonarQube.TeamBuild.PreProcessor/Resources.resx

```diff
@@ -193,37 +196,37 @@ The full path to a settings file can also be supplied. If it is not supplied, th
   <data name="MSG_UpdatingMSBuildTargets" xml:space="preserve">
     <value>Updating build integration targets...</value>
   </data>
-  <data name="SLAP_AdditionalFileAlreadyExists" xml:space="preserve">
+  <data name="RAP_AdditionalFileAlreadyExists" xml:space="preserve">
```

Comment:
Renamed resources from "SLAP_XXX" (_SonarLintAnalyzerProvder_) to "RAP_XXX" (_RoslynAnalyzerProvider_)